core/data/stories.md

```diff
@@ -384,41 +411,63 @@
   - utter_ask_when_tested
 * more
   - utter_maybe_cured
-  - utter_goodbye
+  - utter_ask_anything_else
 
 ## daily ci enroll digression
 * help_pre_existing_conditions
   - utter_explain_pre_existing_conditions
   - utter_ask_pre_existing_conditions_again
-* affirm OR deny OR dont_know
-  - daily_ci_enroll_form
+  - action_revert
```

Comment:
This is risky. If this is predicted by accident, it could revert any user utterance. We should instead use a trigger action and the mapping policy for action_explain_pre_existing_conditions.